apps/dh/api-dh/project.json

```diff
@@ -12,22 +12,6 @@
         "command": "dotnet build apps/dh/api-dh/DataHub.WebApi.sln --configuration=Release"
       }
     },
-    "build-client": {
-      "dependsOn": ["build"],
-      "outputs": ["{workspaceRoot}/libs/dh/shared/domain/src/lib/generated/v1"],
-      "executor": "nx:run-commands",
-      "options": {
-        "commands": [
-          "java -version || echo ❌ Java must be installed",
```

Comment:
praise: Java is not needed anymore! JAVA is NOT needed anymore 😅 😁

nitpick: remove Java from "Prerequisites" section in root readme.

Reply:
Feels good! 😄

dc83290e6b3f63f70949fa9b6adce529dfdeab61